src/lib/media-import/status.ts

```diff
@@ -271,38 +273,90 @@ ${ maybeExitPrompt }
 			void checkStatus( IMPORT_MEDIA_PROGRESS_POLL_INTERVAL );
 		} );
 
+	async function exportFailureDetails(
+		fileErrors: Maybe< AppEnvironmentMediaImportStatusFailureDetailsFileErrors >[]
+	) {
+		const formattedData = buildFileErrors( fileErrors, exportFileErrorsToJson );
+		const errorsFile = `media-import-${ app.name ?? '' }-${ Date.now() }${
+			exportFileErrorsToJson ? '.json' : '.txt'
+		}`;
+		try {
+			await writeFile( errorsFile, formattedData );
+			progressTracker.suffix += `${ chalk.yellow(
+				`⚠️  All errors have been exported to ${ chalk.bold( resolve( errorsFile ) ) }`
+			) }`;
+		} catch ( writeFileErr ) {
+			progressTracker.suffix += `${ chalk.red(
+				`Could not export errors to file\n${ ( writeFileErr as Error ).message }`
+			) }`;
+		}
+	}
+
+	async function fetchFailureDetails( failureDetailsUrl: string ) {
+		progressTracker.suffix += `
+=============================================================
+Downloading errors details from ${ failureDetailsUrl }...
+\n`;
+		try {
+			const response = await fetch( failureDetailsUrl );
+			return ( await response.json() ) as AppEnvironmentMediaImportStatusFailureDetailsFileErrors[];
+		} catch ( err ) {
+			progressTracker.suffix += `${ chalk.red(
+				`Could not download error details\n${ ( err as Error ).message }`
+			) }`;
+			throw err;
+		}
+	}
+
+	async function promptFailureDetailsDownload( failureDetailsUrl: string ) {
+		progressTracker.suffix += `${ chalk.yellow(
+			`⚠️  Error details can be found on ${ chalk.bold(
+				failureDetailsUrl
+			) } ${ chalk.italic.yellow(
+				'(file will be stored in the server for 7 days and its unique link expires in 15 minutes)'
```

Comment:
Is it wise to also print the UTC time of expiry of the link in case the message is not read as it got printed and now the user does not know if 15 minutes have passed?